### PR TITLE
AAVE health alerts

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: src/abi/astETH.json [hunk 1]
```diff
@@ -0,0 +1 @@
...nt256"}],"stateMutability":"view","type":"function"},{"inputs":[{"internalType":"address","name":"recipient","type":"address"},{"internalType":"uint256","name":"amount","type":"uint256"}],"name":"transfer","outputs":[{"internalType":"bool","name":"","type":"bool"}],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"sender","type":"address"},{"internalType":"address","name":"recipient","type":"address"},{"internalType":"uint256","name":"amount","type":"uint256"}],"name":"transferFrom","outputs":[{"internalType":"bool","name":"","type":"bool"}],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"from","type":"address"},{"internalType":"address","name":"to","type":"address"},{"internalType":"uint256","name":"value","type":"uint256"}],"name":"transferOnLiquidation","outputs":[],"stateMutability":"nonpayable","type":"function"},{"inputs":[{"internalType":"address","name":"target","type":"address"},{"internalType":"uint256","name":"amount","type":"uint256"}],"name":"transferUnderlyingTo","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"nonpayable","type":"function"}]
```
Comment (vgorkavenko) [inline src/abi/astETH.json:1]: Does it make sense to use a formatter for abi json files?

File: src/agent-aave.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          description: `variableDebtStETH totalSupply is ${variableDebtStEthTotalSupply.toFixed()}`,
+          alertId: "VARIABLE-DEBT-STETH-SUPPLY",
+          severity: FindingSeverity.High,
+          type: FindingType.Suspicious,
+        })
+      );
+      lastReportedVariableStEthSupply = now;
+    }
+  }
+}
```
Comment (vgorkavenko) [inline src/agent-aave.ts:157]: blank line

File: src/abi/astETH.json (hunk 1 above)
Comment (dgusakov) [reply src/abi/astETH.json:1]: Nope

File: src/agent-aawe.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+import { ethersProvider } from "./ethers";
+
+import LIDO_DAO_ABI from "./abi/LidoDAO.json";
+import ASTETH_ABI from "./abi/astETH.json";
+import STABLE_DEBT_STETH_ABI from "./abi/stableDebtStETH.json";
+import VARIABLE_DEBT_STETH_ABI from "./abi/variableDebtStETH.json";
+
+import {
+  LIDO_DAO_ADDRESS,
+  AAWE_ASTETH_ADDRESS,
```
[the remark this answers is not in the document]
Comment (dgusakov) [reply src/agent-aawe.ts]: Fixed

File: src/agent-aawe.ts [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+      ethersProvider
+    );
+    const astETH = new ethers.Contract(
+      AAWE_ASTETH_ADDRESS,
+      ASTETH_ABI,
+      ethersProvider
+    );
+
+    const astEthBalance = new BigNumber(
+      String(await stETH.functions.balanceOf(AAWE_ASTETH_ADDRESS))
```
[the remark this answers is not in the document]
Comment (dgusakov) [reply src/agent-aawe.ts]: Sounds reasonable. Thx. Fixed

File: src/agent-aave.ts (hunk 2 above)
Comment (dgusakov) [reply src/agent-aave.ts]: Fixed

**Pull request merged**